contracts/libs/0.8.x/BytecodeStorageV1.sol

```diff
@@ -33,7 +37,14 @@ pragma solidity ^0.8.0;
  *      pre-defining return values in some cases in order to simplify need to directly memory manage these
  *      return values.
  */
-library BytecodeStorage {
+
+/**
+ * @title Art Blocks Script Storage Library (Public, Reads)
+ * @author Art Blocks Inc.
+ * @notice The public library for reading from storage contracts. This library is intended to be deployed as a
+ *         standalone contract, and provides all _read_ functionality.
+ */
+library BytecodeStorageReaderV1 {
```

Comment:
nit: I think generally better to keep the "V1" in the file name (specified by importing files), but drop the "V1" off of the library name itself. This simplifies the updates required when we move to a new version to only be import updates.
Same goes for the Writer library in this file

Reply:
Good call – updated here and throughout.